ra_aid/tools/write_file.py

```
@@ -11,14 +11,18 @@
 
 
 @tool
-def write_file_tool(
+def put_complete_file_contents(
     filepath: str, content: str, encoding: str = "utf-8", verbose: bool = True
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)